pkg/reports/bundles/data.go

```diff
@@ -142,8 +142,10 @@ func (d *Data) BuildBundlesQuery() (string, error) {
 		query = query.Limit(uint64(d.Flags.Limit))
 	}
 	if len(d.Flags.Filter) > 0 {
+		query = sq.Select("o.name, o.csv, o.bundlepath, o.version, o.skiprange, o.replaces, o.skips").From(
+			"operatorbundle o, channel_entry c")
 		like := "'%" + d.Flags.Filter + "%'"
-		query = query.Where(fmt.Sprintf("name like %s", like))
+		query = query.Where(fmt.Sprintf("c.operatorbundle_name == o.name AND c.package_name like %s", like))
```

Comment:
is this supposed to be an `ON` clause instead of a `WHERE` clause?

Reply:
@gallettilance I am moving forward here because we need to be able to use this to make it easier we gen the reports for the teams. However, please feel free to let me know if you think that needs to be charged. We can do that in a follow-up.